CSharpOOP2/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿#region Task_1
+using CSharpOOP2;
+
+new Otter();
+
+Otter otterKotaro = new Otter();
+otterKotaro.Name = "Kotaro";
+otterKotaro.Age = 2;
+otterKotaro.Color = "Glossy Brown";
+otterKotaro.IsHungry = false;
+
+Console.WriteLine(otterKotaro.CheckHunger(hoursAfterLastMeal: 2)); 
+
+Otter otterHanna = new Otter
+{
+    Name = "Hanna",
+    Age = 2,
+    Color = "Deep Brown",
+    IsHungry = true
+};
+
+Console.WriteLine(otterHanna.CheckHunger(hoursAfterLastMeal: 3));
+
+Otter otterSeregga = new Otter
+{
```

Comment:
Hi! What is ORIN'KEY?)

The object created using the constructor should have been Seregga, but I think I misunderstood something, since you could not find it